jcli/src/jcli_app/utils/debug_flag.rs

```diff
@@ -9,10 +11,24 @@ pub struct DebugFlag {
 }
 
 impl DebugFlag {
-    pub fn debug_writer(&self) -> Option<impl std::io::Write> {
-        match self.debug {
-            true => Some(std::io::stderr()),
-            false => None,
+    pub fn write_request(&self, request: &Request, body: &RestApiRequestBody) {
+        if !self.debug {
+            return;
+        }
+        eprintln!("{:#?}", request);
+        if body.has_body() {
+            eprintln!("Request body:\n{}", body)
+        }
+    }
+
+    pub fn write_response(&self, response: &RestApiResponse) {
+        if !self.debug {
+            return;
+        }
+        eprintln!("{:#?}", response.response());
```

Comment:
`RestApiResponse::response` is a bit tautological, maybe the method should say that this is the lower level message, like `response.http()`?

Reply:
It's a simple `reqwest::Response` getter, which is why it's `.response()`. I can rename it, but I'm not sure if it's worth the effort.